query/physicalplan/aggregate.go

```diff
@@ -634,3 +617,20 @@ func (a *CountAggregation) Aggregate(pool memory.Allocator, arrs []arrow.Array)
 	}
 	return res.NewArray(), nil
 }
+
+// aggregationHelper is a helper to run the given aggregation function given
+// the set of values. It is aware of the final stage and chooses the aggregation
+// function appropriately.
+func aggregationHelper(
```

Comment:
I think we can probably find a better name for this function. `runAggregation`?

Reply:
Done.